### ISSUE TITLE
No possibility to set empty alt and title

### ISSUE THREAD

**Issue by konhi** [ambassify/react-avatar#248]
Thank you for this great library. I'm happy to use it in my project!
I stumbled upon one little imperfection.

#### Expected behavior
`<Avatar alt="" title="" name="ambassify"` -> `<img alt="" title="" />`

#### Current behavior
`<Avatar alt="" title="" name="ambassify"` -> `<img alt="ambassify" title="ambassify" />`

It's because `"" == false` and it's being checked [here](https://github.com/ambassify/react-avatar/blob/master/src/utils.js#L172)

#### Why is it bad?
It leads to repeated value which is bad for accessibility
```html
<h1>Ambassify</h1>
<img alt="ambassify" title="ambassify"/>
```
"Ambassify" is being repeated **2 times!**

#### Possible non-breaking solution
Add option to pass parameter which will cause in empty alt and title, e.g. `noDescription`, `noAccessibility`, `noAlternativeText`, `noAlt`

`<Avatar alt="" title="" name="ambassify" noDescription` -> `<img alt="" title="" />`

**Comment by JorgenEvens** [comment]
Hi @konhi ,

I would consider the empty string as a non-empty value, and would accept a PR that adds a `arg === ''` check to the line you referenced.

Since your primary issue was that the `alt` and `title` were showing the same value might I suggest another solutions. In cases where it does not have to strictly generate `alt=""` but removing the `alt` attribute entirely works as well, you could set `alt` to `false` rather than the empty string and it would not get rendered into your HTML at all.

**Comment by ashandileo** [comment]
Hi @JorgenEvens 

I can't assign a false value to the title/alt attribute in typescript
I get the error message `Type 'boolean' is not assignable to type 'string'`

<img width="868" alt="Screen Shot 2022-07-21 at 13 24 06" src="https://user-images.githubusercontent.com/61529105/180143786-a0031001-cb67-4aad-b525-c07eec299610.png">

**Comment by JorgenEvens** [comment]
This has been resolved in release [5.0.3](https://github.com/ambassify/react-avatar/releases/tag/v5.0.3). Thanks for the PR @ritchiejacobs !
